tests/test-version.bats

```diff
@@ -3,7 +3,7 @@
 @test "CKAN 2.8.6 still not ready" {
```

Comment:
Do we still need this test?

Reply:
No, is no longer needed. It was removed [here](https://github.com/GSA/catalog.data.gov/pull/147/commits/4080c8926f2102279ed5920c6953d0059bf7e9b4)